doc/source/development/contributing.rst

```diff
@@ -238,22 +238,55 @@ Creating a Python environment (pip)
 If you aren't using conda for your development environment, follow these instructions.
 You'll need to have at least python3.5 installed on your system.
 
-.. code-block:: none
+**Unix**/**Mac OS**
+
+.. code-block:: bash
 
    # Create a virtual environment
    # Use an ENV_DIR of your choice. We'll use ~/virtualenvs/pandas-dev
    # Any parent directories should already exist
    python3 -m venv ~/virtualenvs/pandas-dev
+
    # Activate the virtualenv
    . ~/virtualenvs/pandas-dev/bin/activate
 
    # Install the build dependencies
    python -m pip install -r requirements-dev.txt
 
    # Build and install pandas
-   python setup.py build_ext --inplace -j 4
+   python setup.py build_ext --inplace -j 0
    python -m pip install -e . --no-build-isolation
 
+**Windows**
+
+Below is a brief overview on how to set-up a virtual environment with Powershell
+under Windows. For details please refer to the
+`official virtualenv user guide <https://virtualenv.pypa.io/en/stable/userguide/#activate-script>`__
+
+.. code-block:: powershell
+
+   # Create a virtual environment
+   # Use an ENV_DIR of your choice. We'll use ~\virtualenvs\pandas-dev where
+   # '~' is the folder pointed to by either $env:USERPROFILE (Powershell) or
+   # %USERPROFILE% (cmd.exe) environment variable
+   # Any parent directories should already exist
```

Comment:
I'd move this outside of the code block, it'll be easier to read as a paragraph than as code comments.

Also, since you explain the equivalence of `$env:USERPROFILE` and `%USERPROFILE%` here, you can just leave one of them below. Will look much cleaner and clearer without duplicating the ocmmands.

Other than that, lgtm